src/main/java/pl/mzukowski/ref_exam1/user/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package pl.mzukowski.ref_exam1.user;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.mzukowski.ref_exam1.user.entity.Person;
+import pl.mzukowski.ref_exam1.user.entity.User;
+
+import java.util.Optional;
+
+public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
musi byc public? moze wystarczy tez aby rozszerzal CrudRepository?

Reply:
musi zostać publiczny, faktycznie to repo może rozszerzać tykko cruda